packages/web-app/src/components/appli/Entry/Riggings/Riggings.jsx

```diff
@@ -0,0 +1,76 @@
+// eslint-disable-next-line react/prop-types
+import { useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import { IconButton, Divider, Tooltip } from '@material-ui/core';
+import { useDispatch } from 'react-redux';
+import React, { useState } from 'react';
+import AddCircleIcon from '@material-ui/icons/AddCircle';
+import CancelIcon from '@material-ui/icons/Cancel';
+import { usePermissions } from '../../../../hooks';
+import { postRiggings } from '../../../../actions/Riggings/CreateRigging';
+import CreateRiggingsForm from '../../Form/RiggingsForm/index';
+import ScrollableContent from '../../../common/Layouts/Fixed/ScrollableContent';
+import Rigging from './Rigging';
+import { riggingsType } from '../Provider';
+
+const Riggings = ({ riggings, entranceId }) => {
+  const { formatMessage } = useIntl();
+  const dispatch = useDispatch();
+  const permissions = usePermissions();
+  const [isFormVisible, setIsFormVisible] = useState(false);
+  const onSubmitForm = data => {
+    dispatch(postRiggings(data, entranceId, data.language.id, riggings.id));
+    setIsFormVisible(false);
+  };
+  const isNew = true;
+
+  return (
+    <ScrollableContent
+      dense
+      title={formatMessage({ id: 'Riggings' })}
+      icon={
+        permissions.isAuth && (
+          <Tooltip
+            title={
+              isFormVisible
+                ? formatMessage({ id: 'Cancel' })
+                : formatMessage({ id: 'Add a new riggings' })
+            }>
+            <IconButton
+              color="primary"
+              onClick={() => setIsFormVisible(!isFormVisible)}>
+              {isFormVisible ? <CancelIcon /> : <AddCircleIcon />}
+            </IconButton>
+          </Tooltip>
+        )
+      }
+      content={
+        <>
+          {isFormVisible && (
+            <>
+              <CreateRiggingsForm
+                closeForm={() => setIsFormVisible(false)}
+                isNew={isNew}
+                onSubmit={onSubmitForm}
+              />
+              <Divider />
+            </>
+          )}
+
+          {riggings
+            ?.sort((r1, r2) => r1.title.localeCompare(r2.title))
+            .map((rigging, i) => (
+              <Rigging riggings={riggings} rigging={rigging} index={i} />
```

Comment:
Blocking:
- use the unique id
- why passing "riggings"? Look like it's just for the divider. We could add the divider instead


Not tested:
```suggestion
            .map((rigging) => (
             <>
              <Rigging rigging={rigging} index={rigging.id} />
              {riggings.length - 1 && <Divider variant="middle" />}
             </>
```